packages/ui/package.json

```diff
@@ -93,7 +94,6 @@
     "jsdom": "^16.2.2",
     "lint-staged": "^11.1.2",
     "mini-css-extract-plugin": "^1.3.0",
-    "node-sass": "^5.0.0",
```

Comment:
sure that's safe to remove?

Reply:
We've had a `sass` (dart-sass) and `node-sass`. As I know node-sass is faster, but deprecated from the recent past. `node-sass` doesn't support built-in modules such `sass:math`. 


We use webpack sass-loader plugin which should handle what package should we use. For now, they use both node-sass and sass.

Speaking about rollup, we handle sass with postcss plugin. 

For now, it is a safe remove, because we have this dependency in sass-loader. When it becomes deprecated we won't be needing this (and even handle this).